StayFit/NewPostViewController.m

```diff
@@ -0,0 +1,102 @@
+//
+//  NewPostViewController.m
+//  StayFit
+//
+//  Created by Nikita Agarwal on 7/12/22.
+//
+
+#import "NewPostViewController.h"
+#import "Post.h"
+#import "recipesPost.h"
+#import "CommunityViewController.h"
+
+@interface NewPostViewController ()
+
+@end
+
+@implementation NewPostViewController
+
+- (void)viewDidLoad {
+    [super viewDidLoad];
+}
+
+- (IBAction)shareButton:(id)sender {
+    //takes the user back to the segment that was active after share button is clicked
+    if(self.segoutValue == 0){
+    if(self.image.image && ![self.caption.text isEqualToString:@""]){
+        [Post postUserImage:self.image.image withCaption:self.caption.text withCompletion:^(BOOL succeeded, NSError * _Nullable error) {
+            if (error) {
+                NSLog(@"%s", "could not post");
+            }
+            else{
+                NSLog(@"Awesome");
```

Comment:
i think the text "Awesome" might not be great for a user to understand what happened